permission_handler_platform_interface/lib/src/permission_status.dart

```diff
@@ -119,25 +143,33 @@ extension FuturePermissionStatusGetters on Future<PermissionStatus> {
   /// *Only supported on iOS.*
   Future<bool> get isRestricted async => (await this).isRestricted;
 
+  /// If the permission to the requested feature is permanently denied, the
+  /// permission dialog will not be shown when requesting this permission. The
+  /// user may still change the permission status in the settings.
+  ///
   /// *On Android:*
   /// Android 11+ (API 30+): whether the user denied the permission for a second
   /// time.
   /// Below Android 11 (API 30): whether the user denied access to the requested
   /// feature and selected to never again show a request.
-  /// The user may still change the permission status in the settings.
   ///
   /// *On iOS:*
   /// If the user has denied access to the requested feature.
-  /// The user may still change the permission status in the settings
+  ///
+  /// WARNING: This can only be determined AFTER requesting this permission.
+  /// Therefore make a `request` call first.
```

Comment:
Same as before, since version 11.0.0 this is no longer the case.

Reply:
ack.